wallet-new/src/Cardano/Wallet/Kernel/DB/HdWallet.hs

```diff
@@ -416,14 +416,6 @@ deriveSafeCopy 1 'base ''HdAccountUpToDate
 deriveSafeCopy 1 'base ''HdAccountWithinK
 deriveSafeCopy 1 'base ''HdAccountOutsideK
 
-instance SafeCopy (NewestFirst StrictNonEmpty Checkpoint)  where
-    getCopy = error "TODO: getCopy for (NewestFirst NonEmpty Checkpoint)"
-    putCopy = error "TODO: putCopy for (NewestFirst NonEmpty Checkpoint)"
-
-instance SafeCopy (NewestFirst StrictNonEmpty PartialCheckpoint)  where
-    getCopy = error "TODO: getCopy for (NewestFirst NonEmpty Checkpoint)"
-    putCopy = error "TODO: putCopy for (NewestFirst NonEmpty Checkpoint)"
```

Comment:
? Cleanin up stuff ?

Reply:
Added instances for `StrictNonEmpty` directly (since this is now a type we control).